src/components/button-toggle/README.md

```diff
@@ -0,0 +1,97 @@
+# MdButtonToggle
+
+`MdButtonToggle` is a group of buttons that can be toggled. There are two modes, `multiple` and
+`exclusive`. `Multiple` is a group in which multiple buttons can be toggled at once. `Exclusive` is
+a group in which only one button can be toggled at once.
+
+## `<md-button-toggle>`
+
+### Bound Properties
```

Comment:
I'd put the summary of properties and events at the bottom. It's usually more useful for people to see basic usage examples first.